haiku-apps/alivejournal/alivejournal-1.61.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="Client for LiveJournal online journalling community"
+DESCRIPTION="AliveJournal is a revamped version of Simon Huet's original client."
+HOMEPAGE="http://www.livejournal.com/~alivejournal/"
+COPYRIGHT="2000 Simon Huet-Sean Graham"
```

Comment:
You should split this to make it clean that there are 2 authors. Otherwise some readers might think Simon Huet-Sean Graham is a single person. I'm not sure about the years for Simon Huet, so I just copied your 2000."
```diff
-COPYRIGHT="2000 Simon Huet-Sean Graham"
+COPYRIGHT="
+	2000 Simon Huet
+	2010 Sean Graham
+	"
```